comp110-worksheet-7-tests/UnitTest1.cs

```diff
@@ -8,70 +8,76 @@
 
 namespace comp110_worksheet_7_tests
 {
-	[TestFixture]
-	public class UnitTest1
-	{
-		string c_baseDirectory
-		{
-			get { return Path.Combine(TestContext.CurrentContext.TestDirectory, "..", "..", "pixelcarpack_kenney", "PNG"); }
-		}
+    [TestFixture]
```

Comment:
As a general note, try to avoid committing purely cosmetic changes (e.g. indentation) -- it can make it difficult to pick out the actual changes to code from a diff